drivers/i2c/Kconfig.it8xxx2

```diff
@@ -8,3 +8,16 @@ config I2C_ITE_IT8XXX2
 	  Enable I2C support on it8xxx2_evb.
 	  Supported Speeds: 100kHz, 400kHz and 1MHz.
 	  This driver supports repeated start.
+
+config I2C_ADJUST_SCL_LOW_PRESCALE
+	int "Adjust SCL low period prescale"
```

Comment:
It would be good to have a range attribute for this, as I don't think any integer is useful here is it? At the very least looks to be capped 0 to 255

Reply:
Done.